controllers/player.controller.js

```diff
@@ -1,43 +1,90 @@
 const Player = require('../models/player.model')
 
+const logError = requestId => {
+  console.log(
+    JSON.stringify({
+      kinto_request_id: requestId,
+      error: error
+    })
+  )
+}
+
 exports.hello = (req, res) => {
-  res.send('Hello world')
+  const name = req.params.name
+  res.send(`Hello ${name}`)
 }
 
-exports.player_create = (req, res) => {
-  let player = new Player({
-    name: req.body.name,
-    score: req.body.score
-  })
+exports.player_create = async (req, res) => {
+  try {
+    let player
+    player = await Player.find({ name: req.body.name })
```

Comment:
use `count()` instead of `find()` if you don't care about result

Reply:
I would also rename the variable to `isExisting` or something else that indicates what you wanna check